### DISCUSSION TITLE
Fatture da registrare

### DISCUSSION THREAD

**Discussion by grom62** [fattureincloud/api#251] [Q&A] [answered]
Salve,
utilizzando le API 2 non riesco ad indiviuare e listare le "fatture da registrare".
Ho cercato tra i RECEIVED DOCUMENTS ma non riesco a trovarli.
Qualcuno sa come fare???
Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @grom62 attualmente non esponiamo api per le fatture da registrare.

Grazie,
Emanuele

**Reply by grom62** [reply]
Grazie Emanuele,
spero siano in programma per il futuro.
Ciao.
